### DISCUSSION TITLE
欢迎来到COMP祖传代码库论坛！

### DISCUSSION THREAD

**Discussion by caiqizh** [ZHANG-CAIQI/PolyU_COMP#41] [Announcements]
## 👋 Welcome!
  我们将使用Github的Discussion板块来构造我们的论坛。希望你能：
* 积极发布课程评价
* 提问或者回答问题
* 发布一些Tips
* 能有一些收获，有一段愉悦的体验

### 课程评价
必修课，专业选修课，AMA，SL，CAR等等等等什么课只要是校内的课都可以拿来填写评论！该板块将收集同学对已经上过课程的评奖，给予还在纠结选课的同学一些参考，注意，是参考。同时同学们也可以对自己想选的课程进行提问。在课程评价里，我们加入了Poll功能，因此可以对课程进行评分，让大伙儿的评价更直观！使用规范在[这里](https://github.com/ZHANG-CAIQI/PolyU_COMP/discussions/52)

### Q&A
提问！提问！提问！任何问题都可以问啦，不过和学习或学校相关更好吧。自问自答也是可以的，当然回答都是仅供参考。[使用规范](https://github.com/ZHANG-CAIQI/PolyU_COMP/discussions/53)

### COMP Tips
我们希望大家特别是高届的朋友们在这里发布一些中肯之言，分享自己的课程体验、科研经历、COMP生活等等。当然大家也可以发挥想象，就比如COMP的web服务器，数据库如何使用，有关选课，Health Lifestyle乃至申研等等等等。不过也请大家遵循[使用规范](https://github.com/ZHANG-CAIQI/PolyU_COMP/discussions/54)

**Comment by Yb-Z** [comment]
膜

**Comment by David200308** [comment]
👍
